wiki/frappe_wiki/doctype/wiki_change_request/wiki_change_request.py

```
@@ -711,6 +711,168 @@ def merge_change_request(name: str) -> str:
+			else:
+				merged_items.pop(key, None)
+			continue
+
+		base = normalize_item(base_items.get(key))
+		ours = normalize_item(ours_items.get(key))
+		theirs = normalize_item(theirs_items.get(key))
+		base_content = base_contents.get(key, "")
+		ours_content = ours_contents.get(key, "")
+		theirs_content = theirs_contents.get(key, "")
+
+		result, conflict_type = merge_items(base, ours, theirs, base_content, ours_content, theirs_content)
```

Ruff (0.15.1)

[warning] Unpacked variable `conflict_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)